### ISSUE TITLE
Week 2 Tasks

### ISSUE THREAD

**Issue by marcoplacenti** [stas97/transformers-group-octupus#9]
- [x]  Setup and used Azure to train your model
- [x] Write unit tests for `fetch_data.py`, `make_data.py` and calculate the coverage (@bh1995 @AsgerMunch @sameer-agarwal)
- [x]  use either tensorboard or wandb to log training progress and other important metrics/artifacts in your code
- [x]  remember to comply with good coding practices while doing the project 
- [ ]  Played around with distributed data loading (ensure that it works on cloud @marcoplacenti )
- [x]  (not curriculum) Reformated your code in the pytorch lightning format
- [x]  Model Registration (@marcoplacenti)
- [ ]  Deployed your model using Azure (@marcoplacenti)
- [x]  Checked how robust your model is towards data drifting (@stas97)
- [x]  Deployed your model locally using TorchServe (@stas97)
- [x] Accuracy score logging for tensorboard and printing. Right now it's only loss (@bh1995)
- [x] Write README.md that can say how to run code and other stuff (@bh1995)
- [x] (optional) use a training scheduler (@sameer-agarwal)
- [x] Add some references to the references folder (like Bert model paper) Just for kicks to say we used cookiecutter super dooper well. (@sameer-agarwal)
- [x] (optional) do some comparisons of model runs in a Jupyter Notebook and save some plots in visualizations. (@AsgerMunch)
- [x] Start writing the report (?)

**Comment by stas97** [comment]
- [ ] Accuracy score logging for tensorboard and printing

**Comment by marcoplacenti** [comment]
Yes, I think we can make `predict_model.py` as the client that consumes the model APIs. But I'll have more certain answers later today when I have worked more with it.

We could also have a look at the data drifting thing - so we monitor how and if our model deteriorate in order to trigger a new training to update the mode, or something like that.

**Comment by bh1995** [comment]
Hey guys @stas97 @marcoplacenti @AsgerMunch @sameer-agarwal , is there a meeting going on now? I am having problems opening slack ...

**Comment by sameer-agarwal** [comment]
Yes we are in a meeting

**Comment by stas97** [comment]
https://us02web.zoom.us/j/81942756110?pwd=SzNOSGMvQjVYc1JwdXlVM0lRaUNjQT09

**Comment by sameer-agarwal** [comment]
https://us02web.zoom.us/j/81942756110?pwd=SzNOSGMvQjVYc1JwdXlVM0lRaUNjQT09
